mmselfsup/models/heads/simmim_head.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.runner import BaseModule
+from torch.nn import functional as F
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class SimMIMPretrainHead(BaseModule):
```

Comment:
Better to change to `SimMIMHead` as there is no finetune head (like `MoCoV3Head`)?